info.json

```diff
@@ -0,0 +1,223 @@
+    "publisher": "Fortinet",
+    "description": "A solution framework to manage one or more FortiManagers and attached Fortinet Devices. Build a ZTP Flow that will perform onboarding, lifecycle, and/or offboarding objectives for devices. Create a single pane of glass dashboard for multiple FortiManagers and attached devices. Create reports for devices using jinja to simplify data collection. Interact with external users and/or systems to collect device metadata required for various tasks.",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework",
```

Comment:
> Please provide a link to README.md on the release/1.0.0 branch. i.e
> 
> https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework/blob/release/1.0.0/README.md

Since I don't know the best path here. I will cancel my PR and redo as a 1.0.0 branch request.